src/webextension/widgets/button.js

```diff
@@ -11,6 +11,7 @@ const THEME_CLASS_NAME = {
   primary: `${styles.primaryTheme}`,
   normal: `${styles.normalTheme}`,
   ghost: `${styles.ghostTheme}`,
+  critical: `${styles.criticalTheme}`,
```

Comment:
A minor issue, but I generally see this called "danger" in Mozilla source, instead of "critical". Let's go with that so other Moz folks have an easier time understanding our conventions.